<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>576</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="workspace">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolBox" name="toolBox">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="ICBrowser">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>562</width>
            <height>469</height>
           </rect>
          </property>
          <attribute name="label">
           <string>IC browse</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionLoad_from_file"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>view</string>
    </property>
    <addaction name="actioncpu_load"/>
    <addaction name="actionstremers_throughput"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuview"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionquit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="actionLoad_from_file">
   <property name="text">
    <string>Load from file</string>
   </property>
  </action>
  <action name="actioncpu_load">
   <property name="text">
    <string>cpu load</string>
   </property>
  </action>
  <action name="actionstremers_throughput">
   <property name="text">
    <string>stremers throughput</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionquit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
